docs/bedrock/master/server-configuration.md

```diff
@@ -94,3 +94,12 @@ RewriteRule . /index.php [L]
 ## Managed WordPress hosts and Bedrock
 
 If you're using a [supported WordPress host](deployment.md#supported-wordpress-hosts) such as Kinsta, then contact support and ask them to set your document root to the `web` folder.
+
+If your host uses Apache web server you may create `.htaccess` file at the root of your project with the following content
```

Comment:
To make this more clear, how about something like:

> If your host uses Apache web server and you *can't* change the document root, you can create  an `.htaccess` file at the root of your project with the following content:

Reply:
Looks good to me. Just noticed I paste if statement  right after another one/ Maybe for readability it would be better something like

> Sometimes you can't change the document root on hosted web server. In this case, you can create an `.htaccess` file at the root of your project with the following content:

Reply:
Yep that's even better 👍

Reply:
```suggestion
> Sometimes you can't change the document root on hosted web server. In this case, you can create an `.htaccess` file at the root of your project with the following content:
```